### PR TITLE
Addition of periodic boundary conditions in solver and periodic boundary condition test

### PR DESCRIPTION
**Pull request by atajuelo**

### CODE REVIEW

File: test_fdtd1d.py [hunk 1]
```diff
@@ -65,6 +65,26 @@ def test_fdtd_pmc_conditions():
 
     assert np.corrcoef(solved_e, expected_condition)[0,1] >= 0.99
 
+def test_fdtd_periodic_conditions():
```
Comment (lmdiazangulo) [inline test_fdtd1d.py:68]: This test is checking exactly the same as you were checking with the pmc one. Can you modifiy both, including an initial magnetic field, so the assert is not the same?

**Review by lmdiazangulo — CHANGES_REQUESTED** [review]

**Review by lmdiazangulo — CHANGES_REQUESTED** [review]
improve tests

**Review by lmdiazangulo — CHANGES_REQUESTED** [review]
I think now would pass if it was pec, rather than periodic

**Review by lmdiazangulo — APPROVED** [review]
